pkg/proto/stream/test.proto

```diff
@@ -0,0 +1,43 @@
+// This is only used for tests in this package.  Regenerate with
+//
+//  (cd pkg/proto/stream; protoc --go_out=. --go_opt=paths=source_relative test.proto && mv test.pb.go stream_pb_test.go)
```

Comment:
We have some invoke tasks for protobuf generation; I understand this is for testing, but I think these commands should be added there, see: https://github.com/DataDog/datadog-agent/blob/main/tasks/go.py#L457

Reply:
This got moved out to Molecule, too, and I fitted this into the pre-existing generated code for testing.